translator/source_code/translator_shared_library.py

```diff
@@ -0,0 +1,95 @@
+import xmltodict
```

Comment:
Quite a few of these imports are not used, let's clean it up a bit.

Reply:
cleaned it up.should be fixed